static/app/components/pageTimeRangeSelector.tsx

```diff
@@ -1,74 +1,55 @@
-import {useState} from 'react';
 import styled from '@emotion/styled';
 
 import TimeRangeSelector from 'sentry/components/organizations/timeRangeSelector';
 import {Panel} from 'sentry/components/panels';
-import {DEFAULT_RELATIVE_PERIODS} from 'sentry/constants';
 import {t} from 'sentry/locale';
 import space from 'sentry/styles/space';
-import {defined} from 'sentry/utils';
 
-type Props = React.ComponentProps<typeof TimeRangeSelector> & {
-  className?: string;
-  showPin?: boolean;
-};
-
-function PageTimeRangeSelector({className, customDropdownButton, ...props}: Props) {
-  const [isCalendarOpen, setIsCalendarOpen] = useState(false);
+type Props = React.ComponentProps<typeof TimeRangeSelector> & {className?: string};
 
+function PageTimeRangeSelector({className, ...props}: Props) {
   return (
-    <DropdownDate
-      className={className}
-      isCalendarOpen={isCalendarOpen}
-      hasCustomButton={defined(customDropdownButton)}
-    >
+    <DropdownDate className={className}>
       <TimeRangeSelector
         key={`period:${props.relative}-start:${props.start}-end:${props.end}-utc:${props.utc}-defaultPeriod:${props.defaultPeriod}`}
         label={<DropdownLabel>{t('Date Range:')}</DropdownLabel>}
-        onToggleSelector={isOpen => setIsCalendarOpen(isOpen)}
-        relativeOptions={DEFAULT_RELATIVE_PERIODS}
```

Comment:
not sure why this was necessary. `TimeRangeSelector` already uses `DEFAULT_RELATIVE_PERIODS` if `relativeOptions` is not defined (https://github.com/getsentry/sentry/blob/vl/date-page-filter-panel/static/app/components/organizations/timeRangeSelector/selectorItems.tsx#L22)